lib/import/Importer.js

```diff
@@ -60,22 +69,26 @@ export function importBpmnDiagram(diagram, definitions, bpmnDiagram, done) {
     walker.handleDefinitions(definitions, bpmnDiagram);
   }
 
-  try {
-    importer = diagram.get('bpmnImporter');
-    eventBus = diagram.get('eventBus');
-    translate = diagram.get('translate');
+  return new Promise(function(resolve, reject) {
+    try {
+      importer = diagram.get('bpmnImporter');
+      eventBus = diagram.get('eventBus');
+      translate = diagram.get('translate');
 
-    eventBus.fire('import.render.start', { definitions: definitions });
+      eventBus.fire('import.render.start', { definitions: definitions });
 
-    render(definitions, bpmnDiagram);
+      render(definitions, bpmnDiagram);
 
-    eventBus.fire('import.render.complete', {
-      error: error,
-      warnings: warnings
-    });
-  } catch (e) {
-    error = e;
-  }
+      eventBus.fire('import.render.complete', {
+        error: error,
+        warnings: warnings
+      });
+
+      resolve({ warnings: warnings });
```

Comment:
To match `reject` and as a general good practice always `return resolve(...)`.

Reply:
```suggection
      return resolve({ warnings: warnings });
```

To make it easier to fix :)